src/software/pipeline/main_panoramaPostProcessing.cpp

```diff
@@ -271,7 +281,30 @@ int aliceVision_main(int argc, char** argv)
     oiio::ImageSpec outputSpec(inputSpec);
     outputSpec.tile_width = 0;
     outputSpec.tile_height = 0;
-	outputSpec.attribute("compression", "zip");
+
+    std::string compressionMethod_str = "none";
+
+    if (compressionMethod == image::EImageExrCompression::Auto)
+    {
+        compressionMethod_str = "zip";
+    }
+    else if (compressionMethod != image::EImageExrCompression::None)
+    {
+        compressionMethod_str = EImageExrCompression_enumToString(compressionMethod);
+        if (compressionLevel > 0)
+        {
+            if ((compressionMethod == image::EImageExrCompression::DWAA || compressionMethod == image::EImageExrCompression::DWAB))
+            {
+                compressionMethod_str += ":" + std::to_string(compressionLevel);
+            }
+            else if ((compressionMethod == image::EImageExrCompression::ZIP || compressionMethod == image::EImageExrCompression::ZIPS))
+            {
+                compressionMethod_str += ":" + std::to_string(std::min<int>(compressionLevel, 9));
+            }
+        }
+    }
+	outputSpec.attribute("compression", compressionMethod_str);
+
```

Comment:
Duplicate of write image content, maybe create a function ?

Reply:
Not an exact duplicate (no jpg case) but we could create a function for exr case or a more generic one with a parameter for the image type.